# Header contains sequence and timing information
# from std_msgs/Header
Header header

# Status of camera rail
# True: moving, False: stopped
bool moving

# Distance setting between cameras
# 1: short, 2: medium, 3: long
uint8 baseline
